Automattic/wp-calypso
# Scheduled updates: Configure multisite routes

Related to https://github.com/Automattic/wp-calypso/issues/89473

## Proposed Changes

* Defines `index`, `create`, and `edit` routes for multisite context
* Introduces feature flag: `plugins/multisite-scheduled-updates`
* Renames module folder name

The idea is to keep the same entry route for this feature. On the other side, we will not see the site selector anymore since if you don't provide the siteSlug, it means multisite context.


## Testing Instructions

* Go to `/plugins/scheduled-updates` and check if there is a proper placeholder
* Go to `/plugins/scheduled-updates/create` and check if there is a proper placeholder
* Go to `/plugins/scheduled-updates/edit/random-string-as-schedule-id` and check if there is a proper placeholder
* Pass through the existing site-specific Scheduled Updates feature and check if everything works as before

<img width="783" alt="Screenshot 2024-04-12 at 14 09 01" src="https://github.com/Automattic/wp-calypso/assets/1241413/e2666601-0172-42d4-91f5-fd9604d18e34">
<img width="777" alt="Screenshot 2024-04-12 at 14 09 17" src="https://github.com/Automattic/wp-calypso/assets/1241413/be65e4ac-dea5-43e6-80ed-a3acd53f5a41">
<img width="783" alt="Screenshot 2024-04-12 at 14 09 38" src="https://github.com/Automattic/wp-calypso/assets/1241413/29d2544f-77e7-456f-b908-f0d2910393a3">


## Pre-merge Checklist
